llvm/lib/Target/RISCV/RISCVProcessors.td

```diff
@@ -269,7 +269,8 @@ def SIFIVE_P470 : RISCVProcessorModel<"sifive-p470", SiFiveP400Model,
                                        FeatureUnalignedScalarMem,
                                        FeatureUnalignedVectorMem]),
                                       !listconcat(SiFiveP400TuneFeatures,
-                                                  [TuneNoSinkSplatOperands])>;
+                                                  [TuneNoSinkSplatOperands,
+                                                   TuneOptimizedVectorGather])>;
```

Comment:
I looked back over our internal documentation. I think this might still be profitable on these CPUs. There's an additional constant overhead for LMUL>1 even when the indices only read a single source.